app/src/main/kotlin/at/bitfire/davdroid/resource/LocalAddressBook.kt

```diff
@@ -56,23 +55,21 @@ open class LocalAddressBook @Inject constructor(
         private val logger
             get() = Logger.getGlobal()
 
-        const val USER_DATA_MAIN_ACCOUNT_TYPE = "real_account_type"
-        const val USER_DATA_MAIN_ACCOUNT_NAME = "real_account_name"
         const val USER_DATA_URL = "url"
```

Comment:
If we use the collection ID (good!), can't we just drop the `USER_DATA_URL` and `collectionUrl`?

We should take the

- collection ID to link the address book account with the database,
- collection URL from the DB.